EIPS/eip-4758.md

```diff
@@ -38,10 +38,11 @@ Few applications are affected by this change. The only use that breaks is where
 ## Security Considerations
 
 The following applications of `SELFDESTRUCT` will be broken and applications that use it in this way are not safe anymore:
-1. Any use where `SELFDESTRUCT` is used to burn non-ETH token balances, such as ERC20, inside a contract. We do not know of any such use (since it can easily be done by sending to a burn address this seems an unlikely way to use `SELFDESTRUCT`)
+
+1. Any use where `SELFDESTRUCT` is used to burn non-ETH token balances, such as EIP-20, inside a contract. We do not know of any such use (since it can easily be done by sending to a burn address this seems an unlikely way to use `SELFDESTRUCT`)
```

Comment:
```suggestion
1. Any use where `SELFDESTRUCT` is used to burn non-ETH token balances, such as [EIP-20](/eip-20.md), inside a contract. We do not know of any such use (since it can easily be done by sending to a burn address this seems an unlikely way to use `SELFDESTRUCT`)
```